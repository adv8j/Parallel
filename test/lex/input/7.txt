@TaskGroup g1
{@Properties{@Mem{var1->t1;var2->t2mut;var3->t3 mut;}}}
