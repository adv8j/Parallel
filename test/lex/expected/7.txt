TASKGROUP
IDENTIFIER
LBRACE
PROPERTIES
LBRACE
MEM
LBRACE
IDENTIFIER
ARROW
IDENTIFIER
SEMICOLON
IDENTIFIER
ARROW
IDENTIFIER
SEMICOLON
IDENTIFIER
ARROW
IDENTIFIER
MUT
SEMICOLON
RBRACE
RBRACE
RBRACE
